torchvision/prototype/datapoints/_image.py

```diff
@@ -23,6 +23,11 @@ def __new__(
         device: Optional[Union[torch.device, str, int]] = None,
         requires_grad: Optional[bool] = None,
     ) -> Image:
+        if isinstance(data, PIL.Image.Image):
+            from ..transforms import functional as F
```

Comment:
Nit: I dislike the relative imports over the boundaries of large namespaces

```suggestion
            from torchvision.prototype.transforms import functional as F
```